pandas/tests/io/parser/test_unsupported.py

```diff
@@ -43,23 +45,43 @@ def test_c_engine(self):
 
         # specify C engine with unsupported options (raise)
         with tm.assert_raises_regex(ValueError, msg):
-            read_table(StringIO(data), engine='c',
-                       sep=None, delim_whitespace=False)
+            with tm.assert_produces_warning(FutureWarning):
+                read_table(StringIO(data), engine='c',
+                           sep=None, delim_whitespace=False)
         with tm.assert_raises_regex(ValueError, msg):
-            read_table(StringIO(data), engine='c', sep=r'\s')
+            with tm.assert_produces_warning(FutureWarning):
+                read_table(StringIO(data), engine='c', sep=r'\s')
         with tm.assert_raises_regex(ValueError, msg):
-            read_table(StringIO(data), engine='c', quotechar=chr(128))
+            with tm.assert_produces_warning(FutureWarning):
+                read_table(StringIO(data), engine='c', quotechar=chr(128))
         with tm.assert_raises_regex(ValueError, msg):
-            read_table(StringIO(data), engine='c', skipfooter=1)
+            with tm.assert_produces_warning(FutureWarning):
+                read_table(StringIO(data), engine='c', skipfooter=1)
 
         # specify C-unsupported options without python-unsupported options
         with tm.assert_produces_warning(parsers.ParserWarning):
+            warnings.simplefilter("ignore", category=FutureWarning)
```

Comment:
why are you adding these as read_table? is it to support the read_table deprecation itself? if so, then make a new test function and localize the testing there. if not, then why are you doing it?

Reply:
I wanted to assert that both a `parsers.ParserWarning` and a `FutureWarning` is being emitted, and couldn't figure out any other way than first ignoring the `parsers.ParserWarning`, then ignoring the `FutureWarning`. Should I change the `read_table` calls to `read_csv` calls instead? Or simply ignore the `FutureWarning`?

Reply:
I think you can use a list of the warnings to assert_produces_warning

Reply:
I see, you're right. Maybe it is best to change to `read_csv` anyway, since the tests do not seem to be `read_table` specific?